moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
-	"github.com/micromdm/squirrel/munki/models"
-	"html/template"
-	"io"
-	"log"
-	"net/http"
-	"time"
-)
-
-const STATIC_URL string = "/static/"
-const STATIC_ROOT string = "static/"
-const REPO_PATH string = "/Users/Shared/repo"
-
-type Context struct {
-	Static string
-	Prods  []ParsedPkginfo
-}
-
-// Template expects icon, 'product' name, description, and version, which becomes the download URL
-type ParsedPkginfo struct {
-	Icon        string
-	Name        string
-	Descript    string
```

Comment:
change the field to `Description`. You're not gaining anything by shortening the word and it just loses clarity.